### ISSUE TITLE
pg return error

### ISSUE THREAD

**Issue by simpleblack** [vitaly-t/pg-promise#888]
```
Error: 
  × Return statement is not allowed here
    ╭─[/node_modules/.pnpm/pg@8.11.1/node_modules/pg/lib/crypto/utils.js:4:1]
  4 │ if (useLegacyCrypto) {
  5 │   // We are on an old version of Node.js that requires legacy crypto utilities.
  6 │   module.exports = require('./utils-legacy')
  7 │   return
    ·   ──────
  8 │ }
  9 │ 
 10 │ const nodeCrypto = require('crypto')
    ╰────
```

I am using Next.js version 13.4.12 and pg-promise 11.5.2.
I encountered the following error.
What are the ways to resolve it?

**Comment by vitaly-t** [comment]
I don't see how that issue related to this library. That's a Next.js error, it does not come from `pg-promise`.

**Comment by simpleblack** [comment]
It seems that the issue is resolved in pg version 8.11.2. When can we expect the update from 8.11.1 to 8.11.2?

Thank you.

**Comment by vitaly-t** [comment]
Can you include details? The original issue, when it was introduced, when it was fixed.

**Comment by simpleblack** [comment]
The information is available at the following link.
https://github.com/brianc/node-postgres/issues/2980

**Comment by vitaly-t** [comment]
Released v11.5.3 with the updated driver 8.1.2.

@simpleblack See if the issue has been resolved, and if so - please close the issue ;)

**Comment by simpleblack** [comment]
Good 👍
Thank you.
